Tests/SwiftDocCTests/Infrastructure/SymbolGraph/SymbolGraphLoaderTests.swift

```diff
@@ -141,13 +132,13 @@ class SymbolGraphLoaderTests: XCTestCase {
         try loader.loadAll()
         
         var loadedGraphs = 0
-        var isMainSymbolGraph = false
-        while let graph = try loader.next(isMainSymbolGraph: &isMainSymbolGraph) {
+        
+        for (_, graph) in loader.unifiedGraphs {
             loadedGraphs += 1
-            XCTAssertTrue(isMainSymbolGraph)
```

Comment:
There is no implication. SymbolGraphLoader used to maintain a separate list of unmerged `SymbolGraph`s next to the `UnifiedSymbolGraph`s. However, the `SymbolGraph`s were not used in production code and their aggregation was implemented separately from the aggregation of `UnifiedSymbolGraph`s. I felt like removing the unnecessary `SymbolGraph` aggregation was easier and more maintainable than implementing my changes (transformation, graph merging behavior) for both aggregation processes.

This comes at the cost of having slightly less detailed insight into the loading process. E.g. we now don't know anymore if one symbol graph is a "main" symbol graph, since a `UnifiedSymbolGraph` doesn't really know the concept of extension symbol graphs. I basically removed the assertion because we can't know anymore, but we also don't have to. Indirectly we still kind of know, because if one graph were an extension graph, it would be merged with the respective main graph and we'd get less than 1000 graphs.